hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java

```diff
@@ -72,6 +73,16 @@ public class DatanodeConfiguration {
       Duration.ofMinutes(10).toMillis();
 
   static final boolean CONTAINER_SCHEMA_V3_ENABLED_DEFAULT = false;
+  static final long ROCKSDB_LOG_MAX_FILE_SIZE_DEFAULT = 8 * 1024 * 1024;
+  static final int ROCKSDB_LOG_MAX_FILE_NUM_DEFAULT = 100;
+  static final long ROCKSDB_DELETE_OBSOLETE_FILES_PERIOD_DEFAULT =
```

Comment:
Guessing this is micro seconds
```suggestion
  static final long ROCKSDB_DELETE_OBSOLETE_FILES_PERIOD_MICRO_SECONDS_DEFAULT =
```

Reply:
Yes, it's micro seconds.